lib/engine/step/dividend.rb

```diff
@@ -95,8 +95,8 @@ def payout(entity, revenue)
       end
 
       def dividends_for_entity(entity, holder, per_share)
-        # 1817 2 share half pay uses floats
-        (holder.num_shares_of(entity) * per_share).to_i
+        # 1817 2 share half pay uses floats, for 18MEX num_shares can be a float for NdM
+        (holder.num_shares_of_float(entity) * per_share).ceil
```

Comment:
This won't break 1817, if other games needs to do it differently can subclass.

Reply:
An alternative could be to subclass this method in 18MEX. Do not know how many other titles there are that uses 5% shares?
(Think there are a few, but not that many.)

Anyway, I was considering if there is a situation where the player gets their revenue rounded up, while revenue for the corporation would get rounded down? 
But maybe this is a very rare situation, so a potential 1$ might not have that much effect so it is worth considering?